### ISSUE TITLE
Two apostrophes caused rendering to fail.

### ISSUE THREAD

**Issue by spance** [mathjax/MathJax#2493]
### Issue Summary

Use two apostrophes to represent second derivative, but caused all math elements rendering to fail.

### Steps to Reproduce:

Example:
```html
<!DOCTYPE html>
<head>
<meta charset="utf-8">
<script>
MathJax = {
  tex: {
    inlineMath: [['$', '$'], ['\\(', '\\)']]
  },
  svg: {
    fontCache: 'global'
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js">
</script>
</head>
<body>


<p>$ f' $</p> <!-- OK -->

<p>$ f'' $</p> <!-- Failed -->

<p>$ f^{''} $</p> <!-- Failed -->

</body></html>
```

Error:
```
tex-svg.js:1 Uncaught (in promise) TypeError: t.setAttribute is not a function
    at s.setAttribute (tex-svg.js:1)
    at Q.assistiveMml (tex-svg.js:1)
    at Q.assistiveMml (tex-svg.js:1)
    at e.assistiveMml (tex-svg.js:1)
    at Object.renderDoc (tex-svg.js:1)
    at d.renderDoc (tex-svg.js:1)
    at e.w.render (tex-svg.js:1)
    at tex-svg.js:1
    at Q (tex-svg.js:1)
    at new Promise (<anonymous>)
```

### Technical details:

* MathJax Version: 3.0.5
* Client OS: Win 10 2004
* Browser: Chrome 84.0.4147.105

**Comment by dpvc** [comment]
This is a duplicate of #2402, which is fixed in PR mathjax/MathJax-src#498, part of the coming 3.1 release.

**Comment by dpvc** [comment]
There is a work-around in the original (linked) issue.

**Comment by dpvc** [comment]
Fixed in v3.1 released today.
